Mlem/App/Views/Shared/Toast/ToastModel.swift

```diff
@@ -13,28 +13,28 @@ class ToastModel {
     
     static let main: ToastModel = .init()
     
-    func activeToast(location: ToastLocation) -> Toast? {
-        toasts.first(where: { $0.location == location })
+    func activeToasts(location: ToastLocation) -> [Toast] {
+        Array(toasts.lazy.filter { $0.location == location }.prefix(3))
     }
     
-    func add(_ type: ToastType, location: ToastLocation? = nil, group: String? = nil) {
+    func add(_ type: ToastType, location: ToastLocation? = nil, important: Bool? = nil) {
+        let important = important ?? type.important
         let newToast: Toast = .init(
             type: type,
             location: location ?? type.location,
-            group: group
+            important: important
         )
-        if let group, let index = toasts.firstIndex(where: { $0.group == group }) {
-            toasts[index] = newToast
-        } else {
-            toasts.append(newToast)
+        if !important, let index = toasts.firstIndex(
+            where: { !$0.important && $0.location == location }
```

Comment:
```suggestion
            where: { !$0.important && $0.location == newToast.location }
```

Default `location = nil` results in the `where` clause never being true when location not specified--this should fix it